### PR TITLE
Resolved conflicted HTTP_[...] redeclaration 

### PR DESCRIPTION
**Pull request by nut-code-monkey**
Fix for issue https://github.com/marvinroger/homie-esp8266/issues/120 when ESPAsyncWebServer.h and ESP8266WebServer.h included

### CODE REVIEW

**Comment by me-no-dev** [comment]
I was not aware of this issue :) never tried to run both as well. Maybe we can find a better solution and use the same enum as the regular WebServer

**Comment by etiquet** [comment]
hello, any chance the pull request succeed ?

File: src/AsyncEventSource.cpp [hunk 1]
```diff
@@ -253,7 +253,7 @@ size_t AsyncEventSource::count(){
 }
 
 bool AsyncEventSource::canHandle(AsyncWebServerRequest *request){
-  if(request->method() != HTTP_GET || !request->url().equals(_url))
+  if(request->method() != WebRequest::Method::HTTP_GET || !request->url().equals(_url))
```
Comment (me-no-dev) [inline src/AsyncEventSource.cpp]: sorry have not looked in a long time, but how about it is renamed to WebMethod instead of WebRequest::Method

**Comment by me-no-dev** [comment]
This PR needs to be synced with master in order to merge. Please do read my other comment too.

**Pull request closed without merging**